### PR TITLE
Contribute geo UDFs

### PR DESCRIPTION
**Pull request by afleisc**

### CODE REVIEW

File: udfs/community/azimuth_to_geog_point.sqlx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+config { hasOutput: true }
+/*
+ * Copyright 2019 Google LLC
```
Comment (danieldeleo) [inline udfs/community/azimuth_to_geog_point.sqlx:3]:
```suggestion
 * Copyright 2022 Google LLC
```

Comment (danieldeleo) [reply udfs/community/azimuth_to_geog_point.sqlx:3]: Do this for all others too

File: udfs/community/test_cases.js [hunk 2]
```diff
@@ -1163,3 +1163,59 @@ generate_udf_test("jaccard", [
         expected_output: `CAST(0.25 AS FLOAT64)`
     },
 ]);
+generate_udf_test("knots_to_mph", [
```
Comment (danieldeleo) [inline udfs/community/test_cases.js:1166]: Add tests with NULL inputs

File: udfs/README.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ The following sections cover a few methods of deploying the UDFs.
 
    ```bash
    # Deploy to US
-   gcloud builds submit . --config=deploy.yaml --substitutions _BQ_LOCATION=US
+   gcloud builds submit . --config=deploy.yaml --substitutions _PROJECT_ID=YOUR_PROJECT_ID, _BQ_LOCATION=US
```
Comment (danieldeleo) [inline udfs/README.md:68]:
```suggestion
   gcloud builds submit . --config=deploy.yaml --substitutions _PROJECT_ID=YOUR_PROJECT_ID,_BQ_LOCATION=US
```

**Review by danieldeleo — APPROVED** [review]
